### ISSUE TITLE
Delivery Optimization should be capitalized thus

### ISSUE THREAD

**Issue by jaimeo** [MicrosoftDocs/memdocs#238]
It's the proper name of a feature.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 17ab71ed-269e-8481-ddf7-b1798b4c28de
* Version Independent ID: 4a1da9c8-859f-b6bc-a9c9-870703450c1e
* Content: [Windows 10 Delivery Optimization settings for Intune - Microsoft Intune](https://docs.microsoft.com/en-us/mem/intune/configuration/delivery-optimization-settings#feedback)
* Content Source: [memdocs/intune/configuration/delivery-optimization-settings.md](https://github.com/MicrosoftDocs/memdocs/blob/master/memdocs/intune/configuration/delivery-optimization-settings.md)
* Service: **microsoft-intune**
* Sub-service: **configuration**
* Technology: ****
* GitHub Login: @Brenduns
* Microsoft Alias: **brenduns**

**Comment by Brenduns** [comment]
@jaimeo  Yes, it should be capitalized. Thanks for calling attention to this. I've submitted a content edit that corrects this in our docs.
